easybuild/easyblocks/p/python.py

```diff
@@ -31,7 +31,7 @@
 @author: Pieter De Baets (Ghent University)
 @author: Jens Timmerman (Ghent University)
 """
-
+import copy
```

Comment:
maybe we really should have a style guide. Some easyblocks have a empty line, some don't.